Lib/ntpath.py

```diff
@@ -905,12 +891,13 @@ def commonpath(paths):
 
 try:
     from nt import _path_isdevdrive
+except ImportError:
```

Comment:
Does this change speed anything up?

Reply:
No.

Reply:
Then this change should not be included in this PR